pkg/storage/unified/sql/test/integration_test.go

```diff
@@ -435,7 +435,7 @@ func TestClientServer(t *testing.T) {
 	t.Run("Create a client", func(t *testing.T) {
 		conn, err := grpc.NewClient(svc.GetAddress(), grpc.WithTransportCredentials(insecure.NewCredentials()))
 		require.NoError(t, err)
-		client, err = resource.NewGRPCResourceClient(tracing.NewNoopTracerService(), conn)
+		client, err = resource.NewRemoteResourceClient(tracing.NewNoopTracerService(), conn, authn.GrpcClientConfig{}, true)
```

Comment:
@kalleep FYI this test is disabled at the moment, but if it's ever enabled this will panic, because we're not providing the necessary config fields (`TokenClientConfig` and `TokenRequest`).

That said, to fix this test now that authNZ is enabled by default, we'll need to start the authNZ API server here.

Reply:
Yeah that makes sense.

In order to fix these tests we would need to start configure and start authz (as you mentioned) and have something that can sing our tokens and distribute keys.

It would be really nice to have a simple way to get e2e test like this up and running. I will create an issue for it because it is our stuff that is blocking these kind of test.

Btw I cant ses where this one is skipped?

Reply:
I stand corrected – I glanced over https://github.com/grafana/grafana/blob/4a96f028f3afdbc35734eada148b082e93635481/pkg/storage/unified/sql/test/integration_test.go#L483-L485 and assumed it was always being skipped, but that's not the case!

So I think we need to actually address the test setup here and make sure it passes.

Reply:
Yep, there are some tests that are broken with these changes. Will look into it